server.py

```diff
@@ -0,0 +1,119 @@
+""" Service requests for accessible objects """
```

Comment:
Lets rename this file to what it's main exported class is called - AccessibilityAPIServer